src/ui/layout/AppLayout.vue

```diff
@@ -105,7 +105,13 @@
               class="l-shell__tree"
             />
           </pane>
-          <pane handle="before" label="Recently Viewed" :persist-position="true">
+          <pane
+            handle="before"
+            label="Recently Viewed"
+            :persist-position="true"
```

Comment:
this doesn't need to be bound to anything as it's constant

```suggestion
            persist-position="true"
```

Reply:
Actually, I was wrong about this. If we're passing a boolean, we need the `:v-bind` otherwise it gets passed as a string. _Just Vue 3 Things_